reference/3.0/Microsoft.PowerShell.Utility/Export-PSSession.md

```diff
@@ -253,19 +253,15 @@ Accept wildcard characters: False
 
 ### -Encoding
 
-Specifies the type of encoding for the target file. The default value is **ASCII**.
+Specifies the type of encoding for the target file. The default value is **UTF8**.
```

Comment:
Ensure acceptable values match in YAML to new list.

Reply:
fixed